src/cmd.fs

```diff
@@ -25,8 +25,10 @@ module Cmd =
         cmd |> List.map (fun g -> (fun dispatch -> f >> dispatch) >> g)
 
     /// Aggregate multiple commands
-    let batch (cmds: Cmd<'msg> list) : Cmd<'msg> =
-        List.collect id cmds
+    let batch (cmds: Cmd<'msg> seq) : Cmd<'msg> =
```

Comment:
how about `#seq`, instead of `seq`?

@alfonsogarciacaro from fable perspective, what is the most efficient way to flatten a collection?